libr/bin/format/pyc/marshal.c

```diff
@@ -331,7 +323,7 @@ static pyc_object *get_complex_object(RBuffer *buffer) {
 		return NULL;
 	}
 
-	if ((magic_int & 0xffff) <= 62061) {
+	if ((buffer->magic_int & 0xffff) <= 62061) {
```

Comment:
the magic_int is related to the python header, not the rbuffer struct

Reply:
Is `pyc_object` also related to the python header?  If so, I can add the `magic_int` field there and use `ret->magic_int`.

Reply:
yes, after reading some code in marshal.c i think this field must be stored in there